python/entities.py

```diff
@@ -51,7 +92,7 @@ def intercept_missing_entities(e):
     raise e
 
 
-class Entity(object):
+class Entity(object, metaclass=Meta):
```

Comment:
is this meta-programming???!!!!

Reply:
i would not class defining a metaclass as meta programming. It is at a higher level than usual classes, but it is not at a higher level than the usual program